docs/concepts.md

```diff
@@ -1,25 +1,18 @@
-day 1
-APPROACH 1:
-Collisions array. Tree problem
+### World Position vs Screen Position
+world position is our own coordinate system. (0,0) represents the middle of the intersection. depending on the constants `WORLD_WIDTH` and `WORLD_HEIGHT` the bounds will change.\
+screen position is the describing pixels. (0,0) represents the top left of the window, and depending on `SCREEN_WIDTH` and `SCREEN_HEIGHT` the size of the window will change.
 
-the root of the tree is the current trajectories. The number of collisions represents the number of the node.
-Each leaf of the tree is an adjustment that reduces the number of collisions.
+### Route Position
+vehicles do not have an cartesian x,y position. Instead they have a route position. Route position is a value in meters from 0 - t, where t is the total length of a route.
 
-notes:
-The idea has flaws, for example, it will miss the optimal solution where carA slows down a little and carB speeds up a little. When adjusting carA's speed, it will move it to totally avoid carB.
+benefits:
+- single parameter of displacement (arclength)
+- manager will be working with routes rather than predicting vehicle pos/vel/direction
```

Comment:
I think it might not be immediately obvious why this is a better solution. I think some mention of easier and more efficient collision calculation given one positional parameter would suffice.